foundation-extensions/experience-platform-extension/README.md

```diff
@@ -8,23 +8,23 @@ The Adobe Experience Platform Edge Network extension requires the Identity for E
 
 
 
-## Configure the Edge Network extension in Platform
+## Configure the Edge Network extension in Data Collection UI
```

Comment:
This link is referenced in the Lifecycle documentation. Can you update the link there and double check for any other links using this anchor?
https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/foundation-extensions/lifecycle-for-edge-network/README.md?plain=1#L33